components/higher-order/with-dragging/index.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * WordPress Dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+
+const withDragging = ( OriginalComponent ) => {
+	class Draggable extends Component {
+		constructor() {
+			super( ...arguments );
+
+			this.onDragStart = this.onDragStart.bind( this );
+			this.onDragOver = this.onDragOver.bind( this );
+			this.onDragEnd = this.onDragEnd.bind( this );
+			this.cursorTop = null;
+			this.cursorLeft = null;
+			this.config = {
```

Comment:
Why this needs to be a class property, thinking we could just use module constants for these?

Reply:
Yes, absolutely - wasn't sure how to handle these "config" variables. So local module constants would be ok?